src/node/desktop/src/main/menu-callback.ts

```diff
@@ -333,7 +381,7 @@ export class MenuCallback extends EventEmitter {
     };
 
     if (checkable) {
-      menuItemOpts.type = 'checkbox';
+      menuItemOpts.type = 'radio';
```

Comment:
Leave this as "checkbox". This matches what we do in Qt.

Reply:
Also, remove mention of #10876 from the PR notes. That's a separate work item and isn't implemented by this PR.